.github/workflows/test.yml

```diff
@@ -14,6 +14,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: install
-        run: sudo apt-get install aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils
+        run: sudo apt-get update; sudo apt-get install aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils
```

Comment:
Probably better to join commands with && to avoid install if update failed?

Reply:
I landed code already, let's modify if it cause any issues?

Reply:
you can always push more code ;) it will just cause confusion because logs will show package installation that fails, but it won't be so obvious that it was actually the original apt-get update that failed